tf-psa-crypto/tests/suites/test_suite_psa_crypto.function

```diff
@@ -1234,6 +1234,30 @@ static void interruptible_signverify_get_minmax_completes(uint32_t max_ops,
         *max_completes = 1;
     }
 }
+
+static uint32_t interruptible_key_agreement_get_min_num_ops(size_t key_bits)
+{
+    uint32_t min_values[3][5] = {
+        { 208, 208, 160, 136, 124 }, // P-256
```

Comment:
Please add a comment to explain these values. (It can be a reference to some other place like `ecp.c`.)